x/errors/errors.go

```diff
@@ -106,3 +108,54 @@ func GetInnerInvalidParamsError(err error) error {
 	}
 	return nil
 }
+
+// MultiError is an immutable error that packages a list of errors.
```

Comment:
The alternative is to make it a pointer type, which I'd rather avoid if possible to save some heap allocations and gc.